rust/Cargo.toml

```diff
@@ -24,6 +24,7 @@ regex = "1"
 chrono = "0"
 uuid = { version = "0.8", features = ["serde", "v4"] }
 lazy_static = "1"
+pretty_assertions = "0"
```

Comment:
should move this to dev-dependencies section if it's only used in tests

Reply:
done, thanks